cms/templates/js/show-correctness-editor.underscore

```diff
@@ -0,0 +1,48 @@
+<form>
+<h3 class="modal-section-title" id="show_correctness_label"><%- gettext('Correctness Visibility') %></h3>
+<div class="modal-section-content show-correctness">
+    <ul class="list-fields list-input" role="group" aria-labelledby="show_correctness_label">
```

Comment:
There is no need for the ul/li hierarchy here, and it will cause extra verbosity for screen readers. I know you copied some of these patterns from pre-existing HTML (which I am going to file a bug for), but if the styling doesn't completely break, can you ditch the ul/li stuff and just use straight radio buttons in a group (which implies the correct structure)?